open-reac/README.md

```diff
@@ -1,62 +1,402 @@
 # OpenReac
-OpenReac is a reactive optimal power flow that gives a set of hypotheses for voltage and reactive controls by network equipments such as generators, shunt compensators and transformers. OpenReac can be used for network planning or in operation as well.
+
+OpenReac is a reactive optimal power flow that gives a set of hypotheses
+for voltage and reactive controls by network equipment such as
+generators, shunt compensators and transformers. OpenReac can be used
+for network planning or in operation as well.
+
+---
 
 ## Getting started
-### Knitro
-To run this model, in addition of AMPL you'll need Knitro. Knitro is a proprietary non-linear solver.
 
-Artelys is the company developping Knitro. It is distributing keys [here](https://www.artelys.com/solvers/knitro/).
+### AMPL
+For this project, you must have [AMPL](https://ampl.com/) installed on your machine.
+AMPL is a proprietary tool that works as an optimization modelling language, 
+and it can be interfaced with many solvers.
+
+To run OpenReac, you must add in your `~/.itools/config.yml` an ampl section like this:
+```yaml
+ampl:
+  # Change to the ampl folder path that contains the ampl executable
+  homeDir: /home/user/ampl
+```
 
-After the installation is done and that you got a valid licence, you must have `knitroampl` in your path.
+AMPL is sold by many companies, including Artelys, and you can find keys [here](https://www.artelys.com/solvers/ampl/).
 
-To check, start a bash and run :
+### Non-linear optimization solver
+
+To run the model implemented in AMPL, you'll need a non-linear optimization solver.
+By default, the AMPL code is configured to run Knitro, which is a proprietary non-linear, but you
+are free to configure a different one.
+
+If you chose to run Knitro, you must have `knitroampl` in your path, after the installation
+of the solver is done and that you got a valid licence. To check, start a bash and run:
 ```bash
 knitroampl stub
```

Comment:
This comment, as well as those indicating that Artelys sells keys for AMPL and Knitro were already in the README and come from this commit #1 . 

I also get this result running the command. Despite the error, this indicates that the knitroampl command is recognised. But it could be confusing for the user indeed. 

I propose to remove these comments. What do you think @So-Fras ?